x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_process_detail.tsx

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { memo, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  htmlIdGenerator,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiTextColor,
+  EuiDescriptionList,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import * as event from '../../../../common/endpoint/models/event';
+import { CrumbInfo, formatDate, StyledBreadcrumbs } from './panel_content_utilities';
+import {
+  processPath,
+  processPid,
+  userInfoForProcess,
+  processParentPid,
+  md5HashForProcess,
+  argsForProcess,
+} from '../../models/process_event';
+import { CubeForProcess } from './process_cube_icon';
+import { ResolverEvent } from '../../../../common/endpoint/types';
+import { useResolverTheme } from '../assets';
+
+const StyledDescriptionList = styled(EuiDescriptionList)`
+  &.euiDescriptionList.euiDescriptionList--column dt.euiDescriptionList__title.desc-title {
+    max-width: 8em;
+  }
+`;
+
+/**
+ * A description list view of all the Metadata that goes with a particular process event, like:
+ * Created, Pid, User/Domain, etc.
+ */
+export const ProcessDetails = memo(function ProcessDetails({
+  processEvent,
+  pushToQueryParams,
+}: {
+  processEvent: ResolverEvent;
+  pushToQueryParams: (arg0: CrumbInfo) => unknown;
+}) {
+  const processName = processEvent && event.eventName(processEvent);
+  const processInfoEntry = useMemo(() => {
+    let dateTime = '';
+    const eventTime = processEvent && event.eventTimestamp(processEvent);
+    if (eventTime) {
+      dateTime = formatDate(eventTime);
+    }
+
+    const processInfo = processEvent
+      ? {
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.created', {
+            defaultMessage: 'Created',
+          })]: dateTime,
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.path', {
+            defaultMessage: 'Path',
+          })]: processPath(processEvent),
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.pid', {
+            defaultMessage: 'PID',
+          })]: processPid(processEvent),
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.user', {
+            defaultMessage: 'User',
+          })]: (userInfoForProcess(processEvent) as { name: string; domain: string }).name,
+          [i18n.translate('xpack.securitySolution.enpoint.resolver.panel.processDescList.domain', {
+            defaultMessage: 'Domain',
+            [i18n.translate(
+              'xpack.securitySolution.enpoint.resolver.panel.processDescList.parentPid',

```

Comment:
:reminder_ribbon: add comment to explain better

Reply:
https://github.com/elastic/kibana/pull/69098/commits/532233a3be0f02cdd84a516d0c9235915f1f1240